util/app-config/src/args.rs

```diff
@@ -156,3 +174,23 @@ pub struct MigrateArgs {
     /// check flag present
     pub check: bool,
 }
+
+impl CustomizeSpec {
+    /// No specified parameters for chain spec.
+    pub fn is_unset(&self) -> bool {
+        self.genesis_timestamp.is_none() && self.genesis_message.is_none()
+    }
+
+    /// Generates a vector of key-value pairs.
+    pub fn key_value_pairs(&self) -> Vec<(&'static str, String)> {
+        let mut vec = Vec::new();
+        let genesis_timestamp = self
+            .genesis_timestamp
+            .unwrap_or_else(unix_time_as_millis)
+            .to_string();
+        let genesis_message = self.genesis_message.clone().unwrap_or_else(String::new);
```

Comment:
I want to initialize an identical dev chain in both directories, using the same command with the same `genesis_message` argument, but the resulting chain is different, which is surprising, so I don't think it's appropriate to use the current time as the default value.

Reply:
I agree that we just need to random genesis message by default.

1. Use the timestamp as the genesis message for dev chain by default.
2. User can use `--genesis-message` to customize the message
3. The option `--use-default-spec` is not needed.
4. The option `--genesis-timestamp` is optional. If you really want to customize genesis timestamp, go ahead.